### ISSUE TITLE
Detected that custom integration 'nodered' registers an entity service with a non entity service schema

### ISSUE THREAD

**Issue by vaibhav2912** [zachowj/hass-node-red#296]
### Describe the bug

I am creating a Home Assistant button entity using https://flows.nodered.org/node/node-red-contrib-home-assistant-websocket.

I see a WARNING in the Home Assistant core log after update to Home Assistant Core 2024.9.0

Logger: homeassistant.helpers.frame
Source: helpers/frame.py:184
First occurred: 11:11:23 PM (1 occurrences)
Last logged: 11:11:23 PM

Detected that custom integration 'nodered' registers an entity service with a non entity service schema which will stop working in HA Core 2025.9 at custom_components/nodered/switch.py, line 61: platform.async_register_entity_service(, please create a bug report at https://github.com/zachowj/hass-node-red/issues

### To Reproduce

1. Upgrade Home Assistant to Core 2024.9.0
2. Restart Home Assistant

### Expected behavior

No warning in the log

### Environment Information

Version: 0.65.1

Home Assistant version: 2024.9.0
Companion version: 4.0.2

Node-RED version: 4.0.2
Docker: yes
Add-on: no

Node.js version: v18.20.3 x64 linux
OS: Linux 6.6.46-haos x64

**Comment by RobMeerwijk** [comment]
I have the same error. Entities doen´t show up in HA anymore.

**Comment by RobMeerwijk** [comment]
Does this help? https://developers.home-assistant.io/blog/2024/08/27/entity-service-schema-validation/

**Comment by RobMeerwijk** [comment]
![afbeelding](https://github.com/user-attachments/assets/e90813d0-2cb9-45be-af98-10299c97f274)

**Comment by vaibhav2912** [comment]
@RobMeerwijk,
I don't understand how I can fix this. Can you please elaborate ?
I have a button entity I am registering through nodered for sometime now.
In the issues section, I don't see anything.

![image](https://github.com/user-attachments/assets/72ab02f9-665d-4334-8383-d8b4e6cb51c7)

The entity is still available in HA
| ![image](https://github.com/user-attachments/assets/3eb31012-5c38-4744-9d48-a4517ed1f108) | ![image](https://github.com/user-attachments/assets/c8730ed8-46f7-428c-9c6a-9aadc9068594) |
|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|

**Comment by RobMeerwijk** [comment]
I am investigating all morning, and it is totally unclear. In fact, entities show up after registring in HA. And I can start a flow with a node-red entity ("laden aan") from HA overview. I can see things happening. But the status of "laden aan" remains unavailable. It is on in HA. And the dashboards don't show any values. Wait...

While I am trying to explain, I see that all entities in HA have a _2 added at the end of the entity_id. I can't see why this happenend, but probably, updating 137 nodes will do the trick (on the dashboards as well)

**Comment by RobMeerwijk** [comment]
Everything is working now. I don't know why all entity states got a "_2" suffix. But anyway. Next time I know what to do. Thanks for your reply.

**Comment by RobMeerwijk** [comment]
In order to explain a bit more in detail what happened, see below:

- Yesterday, september 10th I updated HA core to latest version, no problem.
- Today I updated both node-red (4.0.2) and node-red-companion (4.1.0). During this upgrade I saw a message about creating unique identifiers that I ignored (silly mistake in hindsight)

After the upgrade I noticed that the node-red entities were unavailable on the dashboards. I started looking in node red, and there I saw 137 node errors, telling me that enitities were not to be found (clear message in hindsight).

I started by looking in the logs, where I found a seemingly related message about entities. That got me on the wrong track (this issue). It turned out to be pretty simple (the solution, not the cause). 

Somehow the system had decided that all node-red entities should get a _2 suffix in order to be unique. The original entities were all gone after that. 

Changing the entities of all 137 affected nodes and changing the entities on the dashboards solved the problem for now.
